docs/spec/spec-proposals/f1-fee-distribution/f1_fee_distr.tex

```diff
@@ -85,36 +105,61 @@ \subsection{Slashing}
+A malicious validator that gets itself slashed many times would increase the gas to withdraw linearly, but the economic loss of funds due to the slashes should far out-weigh the extra overhead the honest withdrawer must pay for due to the gas.
 
 \subsection{Inflation}
-Inflation is the idea that we want every staked coin to grow in value as time progresses. Each block, every staked token should each be rewarded $x$ staking tokens as inflation, where $x$ is calculated from function which takes state and the block information as input. This also allows for many seemless upgrade's to $x$'s algorithm. This can be added efficiently into the fee distribution model as follows:
+Inflation is the idea that we want every staked coin to create more staking tokens as time progresses. (To the purpose of driving down the worth of unstaked tokens) Each block, every staked token should produce $x$ staking tokens as inflation, where $x$ is calculated from a function $inflation$ which takes state and the block information as input. This inflation can be auto-bonded efficiently into the fee distribution model as follows:
+
+Make each block store the evaluation of $inflation$, which represents the number of staking tokens a single staked token should produce. 
+At the end of each period, increase the validators total number of tokens within the fee distribution by the amount due to inflation. This can be done by multiplying the validators number of staked tokens by $(1 + $ sum of x over the period$)$.
 
-Make each block have an inflation number, by which every staked token should produce $x$ additional staking tokens. In state there is a variable for the sum of all such inflation numbers. Then each period will store this total inflation sum in addition to $\sum_{i=0}^{end}\frac{T_i}{n_i}$. When withdrawing perform a subtraction on the inflation sums at the end and at the start to see how many new staking tokens to produce per staked token. 
+In state a variable should be kept for the sum of all $inflation$ evaluations from now until genesis.
+Then each period will store this total inflation sum in addition to what it already stores per-period.
+The presence of comission rates causes the distribution of delegated stake within a validator to change per block.
+To get around this, when accumulating the value per validator in state, instead of calculating the rewards you'd get for 1 stake, you treat it as though you have $\prod_0^{now} 1 + x$ stake, where $\prod_0^{now} 1 + x$ is the amount of stake accumulated since genesis until now if you had $1$ stake originally bonded.
+When withdrawing, you take the difference as before, which yields the amount of rewards you would have obtained with $(\prod_0^{begin\ bonding\ period}1 + x)$ stake from the block you began bonding at until now. You then divide that by $(\prod_0^{begin\ bonding\ period}1 + x)$ to normalize it, and then proceed as before.
```

Comment:
I don't understand this sentence, do you mean "yields the amount of rewards you would have obtained with $(\prod_0^{begin\ bonding\ period}1 + x)$ stake from block 0 until now"?

Reply:
and by "normalize", do you mean "normalize to $(\prod_n^{now}1 + x)$ stake", where `n` is the block at which you bonded and `now` is when you are withdrawing?